docs/general/kakarot_components.md

```diff
@@ -1,35 +1,32 @@
 # Kakarot Components and Deployment
 
-The entire Kakarot protocol is composed of 3 different kind of Starknet
+The entire Kakarot protocol is composed of 2 different kind of Starknet
 contracts:
 
 - Kakarot
-- Accounts
-  - EOA
-  - Contract account
+- Accounts (EOA & Contract Accounts)
 
 ## Kakarot
 
 The main Kakarot contract is located at:
 [`./src/kakarot/kakarot.cairo`](../../src/kakarot/kakarot.cairo).
 
 This is the core contract which is capable of executing decoded ethereum
-transactions thanks to its `eth_send_transaction` and `eth_call` entrypoint.
+transactions thanks to its `eth_send_transaction` and `eth_call` entrypoint
+(defined in [`./src/kakarot/eth_rpc.cairo`](../../src/kakarot/eth_rpc.cairo)).
 
 Currently, Argent or Braavos accounts contracts don't work with Kakarot.
```

Comment:
actually they work in general, but would fail with account using a cairo_precompile call

Reply:
uhm, wondering if the `get_evm_address` would create an issue, let's say that it's not working for now